<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jsConfigWidget</class>
 <widget class="QWidget" name="jsConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>391</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="jsonConfig">
     <property name="title">
      <string>JSON config</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="jsonIndentLabel">
        <property name="text">
         <string>JSON Indent</string>
        </property>
        <property name="buddy">
         <cstring>jsonIndent</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KIntNumInput" name="jsonIndent">
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="jsonEncodingLabel">
        <property name="text">
         <string>JSON encoding</string>
        </property>
        <property name="buddy">
         <cstring>jsonEncoding</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="jsonEncoding">
        <property name="value" stdset="0">
         <string>utf-8</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="menuConfig">
     <property name="title">
      <string>Menu config</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="lintOnSaveLabel">
        <property name="text">
         <string>Lint when saving</string>
        </property>
        <property name="buddy">
         <cstring>lintOnSave</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="lintOnSave"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="jsAutocompletionLabel">
        <property name="text">
         <string>JavaScript autocompletion</string>
        </property>
        <property name="buddy">
         <cstring>jsAutocompletion</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="jsAutocompletion"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="jQueryAutocompletionLabel">
        <property name="text">
         <string>jQuery autocompletion</string>
        </property>
        <property name="buddy">
         <cstring>jQueryAutocompletion</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="jQueryAutocompletion"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="linterLabel">
        <property name="text">
         <string>Linter</string>
        </property>
        <property name="buddy">
         <cstring>linter</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="linter"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="jQueryReadyConfig">
     <property name="title">
      <string>jQuery ready Snippet</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="jQueryReady"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
